esmvaltool/diag_scripts/blocking/blocking.py

```diff
@@ -0,0 +1,625 @@
+"""Blocking diagnostic"""
+import os
+import logging
+import itertools
+import calendar
+import math
+
+import numpy as np
+from numba import vectorize
+
+import matplotlib
+matplotlib.use('Agg')  # noqa
```

Comment:
This line is no longer needed, as this is now done by the backend via an environmental variable.